### PR TITLE
intermediate exercise

### PR DESCRIPTION
**Pull request by marcofranssen**

### CODE REVIEW

File: beginner/README.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
 # Beginner exercise
+## nginx Deployment
```
Comment (marcofranssen) [inline beginner/README.md]:
```suggestion

## Nginx Deployment
```

File: beginner/README.md [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+
+## Bootstrap sandbox
+
+```bash
+git clone https://github.com/philips-labs/k8s-software-concepts-day.git
+cd k8s-software-concepts-day/beginner
+```
+### Excercises
+
+#### 1. Deploy NGINX deployment
```
Comment (marcofranssen) [inline beginner/README.md]:
```suggestion
#### 1. Deploy Nginx deployment
```

File: beginner/README.md [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
 1. Deploy the nginx deployment from the given yaml in this folder.
-2. Limit the nginx container to use 256MB memory.
-3. Limit the cpu usage to 1 core.
-4. Make sure pod status should be "Ready" in the cluster.
-5. Display the pod status in output of your results.
+2. Fix the resources.
+3. Make sure pod status should be "Ready" in the cluster.
+
+
+#### 2. Deploy NGINX service
```
Comment (marcofranssen) [inline beginner/README.md]:
```suggestion
#### 2. Deploy Nginx service
```

File: beginner/README.md [hunk 4]
```diff
@@ -1,7 +1,38 @@
...
+
+In this exercise we are exposing application via service. We should be able to access application via service
+
+
```
Comment (marcofranssen) [inline beginner/README.md]:
```suggestion
```

File: beginner/nginx-svc.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-service
+spec:
+  type: NodePort
+  # Uncomment the below line to create a Headless Service
```
Comment (marcofranssen) [inline beginner/nginx-svc.yaml]: Shall we remove this comment?

Comment (mandeepgoyat) [reply beginner/nginx-svc.yaml]: comments removed.

**Review by mandeepgoyat — APPROVED** [review]

**Pull request merged**